randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -1127,16 +1155,27 @@ Extra - room_id: [36]
           Any of the following:
               # Get past bomb blocks by jumping
               All of the following:
-                  Can Climb High Jump Wall
                   Any of the following:
+                      # Break Blocks
                       Can Use Power Bombs
                       Screw Attack and Knowledge (Beginner)
+                  Any of the following:
+                      # Get Up
+                      Wall Jump (Intermediate) or Can Climb High Jump Wall
+                      Wall Jump (Beginner) and Can Single Walljump
               # Shinespark from sector shortcuts: https://youtu.be/JChKb_XpMlw
               Level 0 Locks and Speed Booster and Knowledge (Intermediate) and Shinespark Tricks (Beginner) and Disabled Door Lock Rando and Disabled Entrance Rando
-          # Get past pirates
-          Can Kill Tough Beam-Resistant Enemy
-          # Get up to gates and item
-          Space Jump or Wall Jump (Beginner)
+              # Don't do this https://youtu.be/154gQQseqvk
```

Comment:
You're right schwartz, don't do this. Instead, give this a proper comment like "Very precise JBJ" instead.